octoprint_webcam_iframe/static/js/webcam_iframe.js

```diff
@@ -12,32 +12,48 @@ $(function() {
 		self.controlViewModel = parameters[1];
 		self.currentTab = "";
 
+		self.timerHideIframe = null;
+		self.timerHidePluginDashboardIframe = null;
+
 		self.onAllBound = function(allViewModels){
 			$('#webcam_container').replaceWith('<iframe id="webcam_container" src="about:blank" width="588" height="330" style="border: none;" allow="autoplay *; fullscreen *"></iframe>');
 		}
 
 		self.onTabChange = function(current, previous) {
+				var iframe_timeout = self.controlViewModel.settings.webcam_streamTimeout();
```

Comment:
Curious why use `controlViewModel.settings` here rather than `settingsViewmodel.settings` for clarity?

Reply:
No real reason :). Fixed in a new commit to settingsViewmodel.settings + changed the variable name to iframeTimeout to keep it more consistent.